lib/hls/hls_parser.js

```diff
@@ -1202,11 +1228,13 @@ shaka.hls.HlsParser = class {
    * @param {Map.<string, string>} closedCaptions
    * @param {?string} characteristics
    * @param {boolean} forced
+   * @param {boolean} spatialAudio
    * @return {!Promise.<?shaka.hls.HlsParser.StreamInfo>}
    * @private
    */
   async createStreamInfo_(verbatimMediaPlaylistUri, codecs, type, language,
-      primary, name, channelsCount, closedCaptions, characteristics, forced) {
+      primary, name, channelsCount, closedCaptions, characteristics, forced,
+      spatialAudio) {
     // TODO: Refactor, too many parameters
```

Comment:
Haha. Ha.
Nothing to do with your CL, just grieving for the codebase, please ignore this comment.

Reply:
:)